### ISSUE TITLE
Using full screenshots instead of only viewport

### ISSUE THREAD

**Issue by rutgercap** [ServiceNow/BrowserGym#128]
Hi,

My agent uses fullscreen screenshots instead of screenshots of only the viewport. Would it be possible to make this configurable?

I have looked into the codebase and it should be a small fix, I'll add the code change below.
 I wouldn't mind opening a PR for this. The question is if you'd be open to this change and if so, how you'd like to make it configurable. 

Thanks for your time!

file: browsergym/core/src/browsergym/core/observation.py
`
def extract_screenshot(page: playwright.sync_api.Page):
    # Suggested:
    screenshot = page.screenshot(full_page=True, timeout=60000, type="png")
    with io.BytesIO(screenshot) as f:
        img = PIL.Image.open(f)
        img = img.convert(mode="RGB")
        img.save("first.png")

    # Current:
    cdp = page.context.new_cdp_session(page)
    cdp_answer = cdp.send(
        "Page.captureScreenshot",
        {
            "format": "png",
        },
    )
    cdp.detach()

    # bytes of a png file
    png_base64 = cdp_answer["data"]
    png_bytes = base64.b64decode(png_base64)
    with io.BytesIO(png_bytes) as f:
        # load png as a PIL image
        img = PIL.Image.open(f)
        # convert to RGB (3 channels)
        img = img.convert(mode="RGB")
        img = np.array(img)

    return img
`

**Comment by gasse** [comment]
Hi @rutgercap . This would be a nice feature to have. I think the best way to make it configurable is via a flag in the `BrowserEnv()` constructor, like full_page_screenshot or something. Please feel free to open a PR, your contribution is welcome :)

**Comment by rutgercap** [comment]
Great! I'll try to get it done soon. You can expect a PR somewhere this week.
